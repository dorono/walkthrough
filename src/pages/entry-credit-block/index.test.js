import React from 'react';
import {shallow} from 'enzyme';
import {EntryCreditBlockPage} from './index';

const defaultProps = {
    data: {},
    location: {},
};

describe('EntryCreditBlockPage', () => {
    it('should render without errors', () => {
        shallow(<EntryCreditBlockPage {...defaultProps} />);
    });

    it('should match snapshot', () => {
        expect(shallow(<EntryCreditBlockPage {...defaultProps} />)).toMatchSnapshot();
    });
});
