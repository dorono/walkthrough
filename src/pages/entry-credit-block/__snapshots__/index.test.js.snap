// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EntryCreditBlockPage should match snapshot 1`] = `
<div>
  <Container
    primary={true}
    showFullWidthBanner={false}
    title="Entry credit block"
  >
    <Vertical>
      <Box
        key="outline"
        type="outline"
      >
        <Label>
          HASH
        </Label>
        <withRouter(Hash)
          type="ecblock"
        />
      </Box>
      <Box
        key="fill"
        type="fill"
      >
        <Label>
          PARENT DIRECTORY BLOCK
        </Label>
        <DirectoryBlockLink />
      </Box>
    </Vertical>
  </Container>
  <DataLoader
    entriesUrl="/ecblocks/undefined/entries"
    hasLink={false}
    renderContent={[Function]}
  />
</div>
`;
